pdr_backend/lake/etl.py

```diff
@@ -73,25 +73,25 @@ def _drop_temp_sql_tables(self):
             If exists, drop them and rebuild
         """
         # drop the tables if it exists
-        pds = PersistentDataStore(self.ppss.lake_ss.lake_dir)
+        duckDB = DuckDBDataStore(self.ppss.lake_ss.lake_dir)
```

Comment:
I don't quite agree to naming this variable duckDB. It's very confusing and error-prone. The library import for DuckDB is duckdb. This variable name is duckDB. To keep the code readable, we need to have a clear distinction on what we are using: the duckdb library or a custom datastore object?

Also, variable names in Python should always be `snake_case`.

I suggest `duckdb_ds` or `duck_ds` or `duck_datastore` for previous instances of `pds`.

Reply:
just like most other instances, i renamed it simply to "db"